clr/commands.py

```diff
@@ -306,3 +309,7 @@ def arg_text(arg_name, default_value):
         if docstr:
             for line in docstr.split('\n'):
                 print(text_wrapper.fill(line))
+
+    def cmd_argtest(self, a, b, c=4, d=None, e=False):
```

Comment:
Did you mean to commit this testing function?

Reply:
I've added it and removed it enough times I decided to check it in. Might remove later.